src/client/mesh.cpp

```diff
@@ -503,3 +503,18 @@ scene::IMesh* convertNodeboxesToMesh(const std::vector<aabb3f> &boxes,
 	}
 	return dst_mesh;
 }
+
+void setMaterialFilters(video::SMaterialLayer &tex, bool bilinear, bool trilinear, bool anisotropic) {
+	if (trilinear)
+		tex.MinFilter = video::ETMINF_LINEAR_MIPMAP_LINEAR;
+	else if (bilinear)
+		tex.MinFilter = video::ETMINF_LINEAR_MIPMAP_NEAREST;
+	else
+		tex.MinFilter = video::ETMINF_NEAREST_MIPMAP_NEAREST;
```

Comment:
As we have two bool settings anyway, what do you think about changing the meaning of the trilinear setting to "interpolate between mipmaps"?

I.e. here:
```suggestion
	if (bilinear) {
		if (trilinear)
			tex.MinFilter = video::ETMINF_LINEAR_MIPMAP_LINEAR;
		else
			tex.MinFilter = video::ETMINF_LINEAR_MIPMAP_NEAREST;
	} else {
		if (trilinear)
			tex.MinFilter = video::ETMINF_NEAREST_MIPMAP_LINEAR;
		else
			tex.MinFilter = video::ETMINF_NEAREST_MIPMAP_NEAREST;
	}
```

Reply:
I think that would be confusing. If someone enables trilinear filtering, but not bilinear filtering, they should still get trilinear filtering.